.github/workflows/GH-R-CMD-check.yaml

```diff
@@ -65,7 +65,7 @@ jobs:
           try(pak::pkg_install("tidymodels/hardhat"))
           try(pak::pkg_install("tidymodels/modeldata"))
           try(pak::pkg_install("tidymodels/multilevelmod"))
-          try(pak::pkg_install("tidymodels/parsnip"))
+          try(pak::pkg_install("tidymodels/parsnip#991"))
```

Comment:
TODO once the parsnip PR is merged: 

point this back to `main` before merging this PR